docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -127,7 +127,7 @@
     # reduce batch size for distributed training
     batch_size = batch_size // devices
 else:
-    strategy = "auto"  # Set to None if using PyTorch Lightning < 2.0
+    strategy = None  # Set to "auto" if using PyTorch Lightning >= 2.0
     # limit to single device if not using distributed training
```

Comment:
Had to change this back to PyTorch Lightning < 2.0 because other changes are required to make the benchmarks fully compatible with version >= 2.0. See #1112